package.json

```diff
@@ -9,7 +9,7 @@
 		"build-release": "tsc -p src",
 		"prepublish": "safe-publish-latest && npm run build-release",
 		"test": "mocha",
-		"test-published-files": "rm -rf ecmarkup-$npm_package_version.tgz package && npm pack && tar zxvf ecmarkup-$npm_package_version.tgz && cp -r test package/test && cd package && ../node_modules/.bin/mocha",
+		"test-published-files": "rm -rf ecmarkup-$npm_package_version.tgz package && npm pack && tar zxvf ecmarkup-$npm_package_version.tgz && cp -r test package/test && cd package && ../node_modules/.bin/mocha && cd .. && rm -rf ecmarkup-$npm_package_version.tgz package",
```

Comment:
Quote the arguments that contain variables in case, for some reason, `npm_package_version` somehow gets set to `. / .`.

Reply:
Done.

Reply:
You missed one.

Reply:
Already fixed.